modules/tinymce/src/plugins/link/test/ts/browser/ContextToolbarTest.ts

```diff
@@ -43,6 +43,16 @@ describe('browser.tinymce.plugins.link.ContextToolbarTest', () => {
     await UiFinder.pWaitForState<HTMLInputElement>('check link content', SugarBody.body(), '.tox-toolbar input', (ele) => ele.dom.value === 'http://www.google.com');
   });
 
+  it('TINY-8940: toolbar doesn\'t show on non link elements', async () => {
+    const editor = hook.editor();
+    editor.options.set('link_context_toolbar', true);
+    editor.setContent('<p>google</p>');
+    Mouse.trueClickOn(TinyDom.body(editor), 'p');
+    await Waiter.pWait(100);
```

Comment:
```suggestion
    await Waiter.pWait(50);
```
I gave 50 for a specific reason, as that's as high as we like to use normally as it slows done tests if we have too many long waits like this. Additionally, 50 is at least 2 times higher than the 17ms throttle we have in place so it should have given it plenty of time to render.